projects/postgres-assessment.md

```diff
@@ -1,17 +1,17 @@
-## Assessment #9.1: A Quick MongoDB Challenge
+## Assessment #9.1: A Quick PostgreSQL Challenge
```

Comment:
This file was on the list of ignored files in the issue. Please undo these changes.